.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you please delete this file and add a new `.gitignore` file in the root folder and paste this content in the file

```
.vscode
```

This will inform git to ignore any changes in that file which is not needed in the repo.